### DISCUSSION TITLE
Podman issues trying to set up Pi-hole

### DISCUSSION THREAD

**Discussion by fearby** [unifi-utilities/unifi-common#208] [Q&A]
I follow the instructions and put my 10-dns.sh and 20-dns.conflist in the correct folders and runs 10-dns.sh but when I try the last step to run the Pihole container I get:

ERRO[0000] CNI network "dns" not found
ERRO[0000] Error removing timer for container 144d22461455c3e8c878375bd0904fb1eb538ce59a93f46c32aabf08b5e2cd4c healthcheck: unable to get systemd connection to remove healthchecks: dial unix /run/systemd/private: connect: no such file or directory
Error: error configuring network namespace for container 144d22461455c3e8c878375bd0904fb1eb538ce59a93f46c32aabf08b5e2cd4c: CNI network "dns" not found

My 10-dns.sh and 20-dns.conflist are in the correct folders and configured correctly so can't work out what is up.

**Comment by ilkertanli** [comment]
make sure you put a MAC address on 20-dns.conflist:  "mac": "PUT YOUR GENERATED OWN MAC HERE",

{
  "cniVersion": "0.4.0",
  "name": "dns",
  "plugins": [
    {
      "type": "macvlan",
      "mode": "bridge",
      "master": "br5",
      "mac": "PUT YOUR GENERATED OWN MAC HERE",
      "ipam": {
        "type": "static",
        "addresses": [
          {
            "address": "10.0.5.3/24",
            "gateway": "10.0.5.1"
          }
        ],
        "routes": [
          {"dst": "0.0.0.0/0"}
        ]
      }
    }
  ]
}

**Comment by fearby** [comment]
Thanks for the help - got it fixed eventually

**Comment by raymanoz** [comment]
I too am getting this error. I tried updating the mac address.
* can I just create a random mac address for this?
* is the mac address format dash or colon separated?
* Do I also need to update the address and gateway fields to match my new network config?

**Reply by raymanoz** [reply]
The last thing that got it working for me was to `cp /mnt/data/podman/cni/20-dns.conflist  /etc/cni/net.d/`

I saw that someone did this in another post (issues?)

It's working now, but I am not sure if this is the correct solution.
